docs/Hive.AdditionalUserDataExposer/Configuration.md

```diff
@@ -0,0 +1,12 @@
+# Configuration
+
+Hive.AdditionalUserDataExposer exposes user data properties defined by the instance configuration.
+
+## Example
+```jsonc
+"Hive.AdditionalUserDataExposer": {
+    "Properties": [
+        "picture" // Exposes the picture property on the user, which is not serialized with returned user objects by default.
```

Comment:
Should the property be picture or avatar? I'll assume picture since you put it here.

Reply:
Auth0 abstracts all user avatars to `picture` (see [here](https://auth0.com/docs/customize/rules/user-object-in-rules)), so I used that as the example.
![image](https://github.com/Atlas-Rhythm/HiveCorePlugins/assets/41306347/e55da2f9-dd00-480a-8529-349648ccb7c1)